### DISCUSSION TITLE
Keychain prompt

### DISCUSSION THREAD

**Discussion by samuelbradshaw** [Sequel-Ace/Sequel-Ace#668] [Q&A] [answered]
After installing Sequel Ace 3.0.0 (actually, 3.0.1), I got this prompt for Firebase keychain permissions (I'm not sure if any of the numbers needed to be redacted, but I did just in case):

<img width="434" alt="Screen Shot 2020-12-21 at 2 44 27 PM" src="https://user-images.githubusercontent.com/7323052/102825122-677a6b00-439b-11eb-8ffa-ba161da70e55.png">

Is this expected?

**Comment by Jason-Morcos** [comment, marked as answer]
@jamesstout @Kaspik This seems a lil' funky. 
@samuelbradshaw Did you per chance install the beta locally previously? Particularly 3.0.x beta

**Reply by Jason-Morcos** [reply]
My hunch is you previously installed the beta and this is something about the non-beta version requesting access to stuff generated by the beta.
Which shouldn't happen with the app group/keychain group, but perhaps something is funky with those?

**Reply by samuelbradshaw** [reply]
I did briefly a few weeks ago to check out the new icons, but I didn't keep it installed. Do you think there might be some files or keychain entries lingering from that?

**Reply by samuelbradshaw** [reply]
I'm still on macOS Catalina (10.15.7), if that makes a difference. (Still the old icons because Catalina doesn't have SF Symbols, so my installing the beta to look at the icons wasn't very exciting. :) )

**Reply by Jason-Morcos** [reply]
@samuelbradshaw I would go ahead and open an issue for this as it does look like a bug. That being said, not particularly high priority if all it is is asking for your password to unlock the keychain (regardless of beta) but may indicate something funky with app/keychain groups or keychain permissions

**Reply by Kaspik** [reply]
Even non-Big Sur has new icons tho! :)))

I experienced this too, but I didn't really care - we do use keychain, so I thought it simply asks me to use it in order to access also the beta identifier created password. Not sure why Firebase would be mentioned there ....

**Reply by samuelbradshaw** [reply]
You're right – I opened Keychain Access and found the entry – it was added on December 1st, and Keychain Access shows that Sequel Ace beta has access to it, so it must have been added by the beta.
<img width="558" alt="Screen Shot 2020-12-21 at 2 59 37 PM" src="https://user-images.githubusercontent.com/7323052/102826035-271bec80-439d-11eb-9a3a-774822dc28b8.png">

I guess I'll delete the old keychain entry to avoid anything potentially unfinished lingering from the pre-release build. :)

**Reply by Jason-Morcos** [reply]
Shouldn't be dangerous at all to have that keychain issue! I just think it's weird that it's asking you for access at all - the whole point of app groups/keychain groups is to be able to share the keychain between multiple apps seamlessly, in our case the beta and main apps

**Reply by Jason-Morcos** [reply]
I wonder if we're not actually using the keychain group... lol

**Reply by Kaspik** [reply]
The incorrect prompt has been fixed in 3.0.2 I believe.
